multiview/embed/gcca.py

```diff
@@ -188,62 +226,63 @@ def fit(
         """
-        Embeds data matrix(s) using the fitted projection matrices
+        Embeds data matrix(s) using the fitted projection matrices. May be
+        used for out-of-sample embeddings.
 
         Parameters
         ----------
         Xs: list of array-likes
             - Xs shape: (n_views,)
             - Xs[i] shape: (n_samples, n_features_i)
-            The data to embed based on the prior fit function. If
-            view_idx defined, Xs is 2D, single view
-        view_idx: int
-            The index of the view whose projection to use on Xs.
-            For transforming a single view inpu.
+            A list of data matrices from each view to transform based on the
+            prior fit function. If view_idx defined, then Xs is a 2D data
+            matrix corresponding to a single view.
+        view_idx: int, default=None
+            For transformation of a single view. If not None, then Xs is 2D
+            and views_idx specifies the index of the view from which Xs comes
+            from.
 
         Returns
         -------
-        Xs_transformed : array-like 2D
-            if view_idx not None, shape same as Xs
+        Xs_transformed : list of array-likes or array-like
+            Same shape as Xs
         """
-        if self._projection_mats is None:
+        if self.projection_mats_ is None:
             raise RuntimeError("Must call fit function before transform")
         Xs = check_Xs(Xs)
         if view_idx is not None:
-            return self._center(Xs[0]) @ self._projection_mats[view_idx]
+            return self.center(Xs[0]) @ self.projection_mats_[view_idx]
         else:
             return np.array(
                 [
-                    self._center(x) @ proj
-                    for x, proj in zip(Xs, self._projection_mats)
+                    self.center(x) @ proj
+                    for x, proj in zip(Xs, self.projection_mats_)
                 ]
             )
 
-    def fit_transform(self, Xs, **fit_params):
+    def fit_transform(self, Xs):
         """
-        Fit to data, then transform it.
-
-        Fits transformer to Xs optional parameters fit_params and returns a
-        transformed version of the Xs.
+        Fits transformer to Xs and returns a transformed version of the Xs.
 
         Parameters
         ----------
         Xs: list of array-likes
             - Xs shape: (n_views,)
             - Xs[i] shape: (n_samples, n_features_i)
-            The data to fit to. Each sample will receive its own embedding.
+            The data to fit to. Each sample will receive its own
+            transformation matrix and projection.
 
         Returns
         -------
         Xs_transformed : array-like 2D if view_idx not None, otherwise
-            (n_views, n_samples, n_components)
+            (n_views, n_samples, self.n_components)
         """
 
-        return self.fit(Xs, **fit_params).transform(Xs)
+        return self.fit(Xs).transform(Xs)
```

Comment:
very small comment, but this could be in base class

Reply:
See #47